apps/www/_blog/2023-08-12-vecs-adapters.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Vecs Adapters'
+description: 'supabase/vecs adapters: query vectors with text'
+author: oli_rice
+image: 2023-06-01-vecs/og-image.jpg
+thumb: 2023-06-01-vecs/og-image.jpg
+tags:
+  - postgres
+  - AI
+  - python
+date: '2023-08-11'
+toc_depth: 3
+---
+
+[supabase/vecs](https://github.com/supabase/vecs) is our python client for rapid development of vector projects based on postgres + pgvector.
+
+Today, we're releasing a new feature for vecs called "adapters" that enables developers to manage embeddings without ever having to touch a vector.
+Adapters allow you to associate transforms with a collection (table of vectors) that are applied to your input when upserting and querying.
```

Comment:
this sentence is a bit opaque, perhaps something like:

"Adapters transform your input into a new format when upserting and querying. For example, you can transform text into embeddings, or ...."